File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/sykefraværshistorikk/aggregert/AggregertStatistikkDto.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.sykefraværshistorikk.aggregert;

import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AggregertStatistikkDto {

    public List<StatistikkDto> prosentSiste4KvartalerTotalt = List.of();
    public List<StatistikkDto> prosentSiste4KvartalerGradert = List.of();
    public List<StatistikkDto> prosentSiste4KvartalerKorttid = List.of();
    public List<StatistikkDto> prosentSiste4KvartalerLangtid = List.of();

    public List<StatistikkDto> trendTotalt = List.of();

    public List<StatistikkDto> tapteDagsverkTotalt = List.of();
    public List<StatistikkDto> muligeDagsverkTotalt = List.of();
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/sykefraværshistorikk/aggregert/AggregertStatistikkDto.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.sykefraværshistorikk.aggregert

data class AggregertStatistikkDto(
    val prosentSiste4KvartalerTotalt: List<StatistikkDto> = listOf(),
    val prosentSiste4KvartalerGradert: List<StatistikkDto> = listOf(),
    val prosentSiste4KvartalerKorttid: List<StatistikkDto> = listOf(),
    val prosentSiste4KvartalerLangtid: List<StatistikkDto> = listOf(),
    val trendTotalt: List<StatistikkDto> = listOf(),
    val tapteDagsverkTotalt: List<StatistikkDto> = listOf(),
    val muligeDagsverkTotalt: List<StatistikkDto> = listOf(),
)